master/lustrefactory.py

```diff
@@ -72,7 +80,62 @@ def buildCommand(props):
 
     return args
 
-def getBuildFactory(gerrit_repo, **kwargs):
+def createTarballFactory(gerrit_repo):
+    """ Generates a build factory for a tarball generating builder.
```

Comment:
You should follow the Docstrings guidelines here:

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

And in that pep in general.  But since this is just the configuration, I won't hold up the patch on that.